fun_certificates/tests/test_certificates.py

```diff
@@ -65,3 +67,9 @@ def test_certif_not_verified(self):
         self.certif.save()
         response = self.client.get(reverse('short-cert-url', args=[self.encoded]))
         self.assertEqual(404, response.status_code)
+
+    def test_works_if_delivery_month_has_accent(self):
+        august = datetime.datetime(2016, 8, 1, 14, 33, 46, 241780) # August the 1st, the month is accentuated in French
+        self.certif.modified_date = august
```

Comment:
I suspect this does not test what you want (you need `self.certif.save()` to take the date into account).